schema/test/unit/computed_columns/form_change_holdback_amount_to_date_test.sql

```diff
@@ -46,44 +50,103 @@ values (
   3,
   1,
   'milestone',
-  '{"reportType": "General Milestone", "hasExpenses": true, "reportDueDate": "2022-11-10 15:09:36.264005-08", "maximumAmount": 50000, "reportingRequirementIndex": 4, "certifierProfessionalDesignation": "Professional Engineer"}'
-),
-(
+  '{"reportType": "General Milestone", "hasExpenses": true, "reportDueDate": "2022-11-14 15:09:36.264005-08", "calculatedHoldbackAmount": 10000, "adjustedHoldbackAmount": 20000, "reportingRequirementIndex": 1, "certifierProfessionalDesignation": "Professional Engineer"}'
+);
+/** SETUP END **/
+
+
+select is(
+  (
+    with record as (
+    select row(form_change.*)::cif.form_change
+    from cif.form_change where id=3
+    ) select cif.form_change_holdback_amount_to_date((select * from record))
+  ),
+  (
+    70000::numeric
+  ),
+  'Returns the correct sum of all milestones with expenses'
+);
+
+
+-- adjustedHoldbackAmount or calculatedHoldbackAmount are NULL
+insert into cif.form_change(
+  id,
+  operation,
+  form_data_schema_name,
+  form_data_table_name,
+  form_data_record_id,
+  project_revision_id,
+  json_schema_name,
+  new_form_data
+) overriding system value
+values (
   4,
   'create',
   'cif',
   'reporting_requirement',
-  3,
-  1,
+  4,
+  2,
   'milestone',
-  '{"reportType": "Performance Milestone", "hasExpenses": false, "reportDueDate": "2022-11-10 15:09:36.264005-08", "adjustedHoldbackAmount": 20000, "reportingRequirementIndex": 4, "certifierProfessionalDesignation": "Professional Engineer"}'
+  '{"reportType": "General Milestone", "hasExpenses": true, "reportDueDate": "2022-11-14 15:09:36.264005-08", "adjustedHoldbackAmount": 20000, "reportingRequirementIndex": 1, "certifierProfessionalDesignation": "Professional Engineer"}'
 ),
 (
   5,
   'create',
   'cif',
-  'funding_parameter',
-  3,
-  1,
-  'funding_parameter_EP',
-  '{"holdbackPercentage": 10}'
+  'reporting_requirement',
+  5,
+  2,
+  'milestone',
+  '{"reportType": "General Milestone", "hasExpenses": true, "reportDueDate": "2022-11-14 15:09:36.264005-08", "calculatedHoldbackAmount": 10000, "reportingRequirementIndex": 1, "certifierProfessionalDesignation": "Professional Engineer"}'
+);
+select is(
+  (
+    with record as (
+    select row(form_change.*)::cif.form_change
+    from cif.form_change where id=4
+    ) select cif.form_change_holdback_amount_to_date((select * from record))
+  ),
+  (
+  30000::numeric
+  ),
+  'Returns the correct value when adjustedHoldbackAmount or calculatedHoldbackAmount exists'
 );
```

Comment:
This seems to be almost the same test as the first one

Reply:
The test is similar, but the data being tested is different since one milestone is missing a calculated value and the other is missing an adjusted value